src/lldp_syncd/daemon.py

```diff
@@ -188,21 +188,28 @@ def parse_update(self, lldp_json):
 
     def parse_chassis(self, chassis_attributes):
         try:
-            (rem_name, rem_attributes), = chassis_attributes.items()
-            chassis_id_subtype = str(self.ChassisIdSubtypeMap[rem_attributes['id']['type']].value)
-            chassis_id = rem_attributes['id']['value']
+            (rem_name, rem_attributes) = chassis_attributes.items()[0]
```

Comment:
We don't know the order in the dictionary. So [0] will not work.
It's better to check like
``` if 'id' In chassis_attributes.keys() ```